nexus/db-model/src/inventory.rs

```diff
@@ -158,6 +158,36 @@ impl From<HwRotSlot> for RotSlot {
     }
 }
 
+// See [`M2Slot`].
+impl_enum_type!(
+    HwHostPhase1SlotEnum:
+
+    #[derive(Copy, Clone, Debug, AsExpression, FromSqlRow, PartialEq)]
+    pub enum HwHostPhase1Slot;
```

Comment:
Thoughts on just calling it something like `M2Slot`?

Reply:
Renamed to `HwM2Slot` in bdbd14d22